boolean/test_boolean.py

```diff
@@ -760,6 +760,28 @@ def test_simplify(self):
         )
         assert result.pretty() == expected.pretty()
 
+    def test_absorption_invariant_to_order(self):
+        algebra = BooleanAlgebra()
+
+        a, b = algebra.symbols(*"ab")
+
+        e = (~a | ~b) & b & ~a
+        args = [
+            ~a | ~b,
+            ~a,
+            b,
+        ]
+
+        result_original = e.absorb(args)
+
+        args[1], args[2] = args[2], args[1]
```

Comment:
Could there be a way to make this more readable?

Reply:
Your call if any of these are more or less readable:
```suggestion
        args[1:3] = reversed(args[1:3])
```

```suggestion
        args[1:3] = args[2:0:-1]
```

```suggestion
        args = [args[0], *reversed(args[1:3])]
```

I suppose there's always plain-old...
```suggestion
        args = [args[0], args[2], args[1]]
```

Reply:
Good point.